src/icons/fileIcons.ts

```diff
@@ -654,7 +654,7 @@ export const fileIcons: FileIcons = {
         { name: 'fusebox', fileNames: ['fuse.js'] },
         { name: 'heroku', fileNames: ['procfile', 'procfile.windows'] },
         { name: 'editorconfig', fileNames: ['.editorconfig'] },
-        { name: 'gitlab', fileNames: ['.gitlab-ci.yml'] },
+        { name: 'gitlab', fileExtensions: ['.gitlab-ci.yml'] },
```

Comment:
Can you please remove the leading dot before "gitlab"? Otherwise it's not working anymore in the `fileExtensions` property.

Reply:
[okay](https://github.com/PKief/vscode-material-icon-theme/pull/465/commits/c8daf32122dd859c1fb78702e0051b45ed68e850)